content/influxdb/cloud/query-data/parameterized-queries.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Use parameterized Flux queries
+description: >
+  Use parameterized queries to re-use queries and prevent injection attacks
+weight: 104
+menu:
+  influxdb_cloud:
+    name: Parameterized queries
+    parent: Query data
+influxdb/cloud/tags: [query, security]
+---
+
+Flux supports parameterized queries in InfluxDB Cloud.
+Parameterized queries make Flux queries more reusable, and can also be used to help prevent injection attacks
+
+Use parameterized queries to dynamically change values used in a query without having to re-write it.
+This feature allows users to define the values of variables in a separate field in a request payload.
+A parameterized query enables you to supply arguments which are then inserted into the Flux query for it to be executed.
+
+Use parameterized queries when executing Flux with untrusted user input;
+for example, in a web or IoT application.
+For more information on security and query parameterization,
+see the OWASP [SQL Injection Prevention Cheat Sheet][].
+While this guide is about SQL, it contains useful general advice.
```

Comment:
Sometimes it's hard to tell if there's text after a long Markdown link in my editor. I've been thinking about using reference style a bit more often. But we can avoid it if it'll confuse us or if people don't like it :)

Reply:
I'm not against it, we just have to make sure the refs don't feel out of place. Right now, it's just a random link at the bottom of the page. Put an `h5` or `h6` heading above the link ref – `References` or something similar. Avoid `Related` since we already have a related links section that gets added with frontmatter.